gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -150,7 +150,7 @@ static String encodeMetadataValues(byte[] bytes) {
   }
 
   @Nullable
-  private static byte[] decodeMetadataValues(String value) {
+  static byte[] decodeMetadataValues(String value) {
```

Comment:
why making this public?

Reply:
I was utilising it in one of the test. Ideally it should be in some util class. I understand it can create issue being a method available to be used via `gcsImpl` class. Will refactor it.